pages/Documents/Reporting/AgentActivityAPI/Methods/get-status-changes.md

```diff
@@ -34,11 +35,13 @@ Note that this affects the limit and offset parameters (in v1 they refer to an a
 | to | This filters the results to status changes occurred within the timeframe between `from` and `to`  | [RFC 3339](https://tools.ietf.org/html/rfc3339) date-time string | Optional | request time |{::nomarkdown}<ul> <li>If provided, `from` must also be provided</li> <li>Maximum timeframe between `from` and `to`: 1 month</li> </ul>{:/}|
 | agentId | This filters the results to status changes of the agent with the specified LivePerson ID | number | Optional | |{::nomarkdown}<ul> <li>If provided, `empId` must not be provided</li> <li>If neither `agentId` nor `empId` are provided, all agents will be returned</li> </ul>{:/}|
 | empId | This filters the results to status changes of the employee with the specified employee ID | string | Optional | |{::nomarkdown}<ul> <li>If provided, `agentId` must not be provided</li> <li>If neither `agentId` nor `empId` are provided, all agents will be returned</li> </ul>{:/}|
-| limit | This limits the number of agents, for which status changes will be included in the results | number | Optional | 50 | **v1**: limit the number of agents <br> **v2**: limit the number of state changes records |
-| offset | This allows to get more results in case you have more agents than `limit` | number | Optional | 0 | **v1**: refer to the number of agents <br> **v2**: refer to the number of state changes records |
+| limit | This limits the number of records in the response to this value | number | Optional | V1: 50 Agents V2: 1000 state changes | Maximum value: V1: 100 <br/> Agents V2: 5000 state changes |
```

Comment:
Entire file: switch to simple syntax (`<br>`)—no need to close the `br` elements

Reply:
@j9t Removed the </br>